src/api/app/components/request_decision_component.html.haml

```diff
@@ -49,21 +46,27 @@
             .form-check.mb-2.mt-2.d-none
               = check_box_tag('add_submitter_as_maintainer_0', "#{@action.target_project}_#_#{@action.target_package}",
                               false, class: 'form-check-input')
-          -# Accept buttons
-          %span.dropdown.me-2
-            = button_tag('Accept', type: 'button', role: 'button', id: 'decision-buttons-group', class: 'btn btn-secondary dropdown-toggle',
-                          data: { 'bs-toggle': 'dropdown'}, aria: { 'haspopup': 'true', 'expanded': 'false' })
-            .dropdown-menu{ aria: { labelledby: 'decision-buttons-group' } }
-              -# Simple Accept request submit
-              = submit_tag('Accept request', name: 'accepted', class: 'btn-link dropdown-item', data: confirmation,
-                            title: 'Accept the request, this will apply the changes on the target.')
-              -# Accept and make maintainer submit
-              - if accept_with_options_allowed? && show_add_submitter_as_maintainer_option?
-                = submit_tag("Accept and make #{@creator} maintainer of #{make_maintainer_of}",
-                            name: 'accepted',
-                            class: 'btn-link dropdown-item', data: confirmation,
-                            title: 'Accept the request. this will apply the changes on the target.',
-                            id: 'accept-and-make-maintainer')
+          - if accept_with_options_allowed? && show_add_submitter_as_maintainer_option? && @forward_allowed
+            = render 'webui/request/accept_dropdown',
+              make_maintainer_option: true,
+              forward_request_option: true,
+              confirmation: confirmation,
+              creator: @creator,
+              target: target
```

Comment:
This is not how you render partials or pass options to them...

Reply:
Why not? What's wrong?

Reply:
`render partial: 'name', locals: {}`

Reply:
> `render partial: 'name', locals: {}`

You can shortcut it as `render 'name', {}`